ACadSharp/IO/DWG/DwgObjectSectionReader.cs

```diff
@@ -170,30 +176,30 @@ private ObjectType getEntityType(long offset)
 				ulong handleSectionOffset = (ulong)this._crcReader.PositionInBits() + sizeInBits - handleSize;
 
 				//Create a handler section reader
-				this._objectReader = DwgStreamReader.GetStreamHandler(this._version, new StreamIO(this._crcStream, true).Stream);
+				this._objectReader = DwgStreamReader.GetStreamHandler(this._version, new StreamIO(this._crcStreamBuffer).Stream);
```

Comment:
Instead of using the `StreamIO` class, change it for `MemoryStream`, it seems to work fine and it will avoid some issues.

`StreamIO` is an old class from a shared project and needs some re-thinking.